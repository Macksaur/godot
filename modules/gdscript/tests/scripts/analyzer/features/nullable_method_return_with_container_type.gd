func test():
    print(_get_result())


func _get_result() -> Array[int]:
    var might_be_null: Array[int?] = []
    return might_be_null

