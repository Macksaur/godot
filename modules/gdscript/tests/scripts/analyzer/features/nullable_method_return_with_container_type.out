GDTEST_ANALYZER_ERROR
Cannot return nullable value of type "Array[int?]" because the function return type is "Array[int]".
